app/js/lib/foxbox/settings.js

```diff
@@ -203,4 +187,33 @@ export default class Settings extends EventDispatcher {
   get apiVersion() {
     return API_VERSION;
   }
+
+  clear() {
+    return new Promise((resolve) => {
+      this.session = null;
+      // @todo Remove only the items set here.
```

Comment:
Yeah, it's more complex than I expected. But I guess we have to accept that. I think there shouldn't be any side effect to using this lib, like clearing all of the local storage. The view may want to use the local storage too.